### PR TITLE
Update documentation scheduler

### PR DESCRIPTION
**Pull request by jpbruinsslot**
### Changes

Minor updates to the scheduler documentation

### CODE REVIEW

File: mula/docs/architecture.md [hunk 1]
```diff
@@ -3,28 +3,28 @@
 ## Purpose
 
 The *scheduler* is tasked with populating and maintaining a priority queue of
-items that are ranked, and can be popped off (through api calls).
+items that are ranked, and can be popped off through http api calls.
```
Comment (ammar92) [inline mula/docs/architecture.md:6]:
```suggestion
items that are ranked, and can be popped off through HTTP API calls.
```

**Review by ammar92 — APPROVED** [review]
Looks good to me 👍 Just a minor suggestion. Please also check the pre-commit suggestion

**Comment by jpbruinsslot** [comment]
> Looks good to me 👍 Just a minor suggestion. Please also check the pre-commit suggestion

yeah had problems setting it up on my laptop

**Comment by ammar92** [comment]
Please one more update so we can merge asap

**Review by ammar92 — APPROVED** [review]

**Review by dekkers — APPROVED** [review]

**Pull request merged**